scripts/lint_site.sh

```diff
@@ -167,7 +167,7 @@ if [[ ${SKIP_LINK_CHECK:-} != "true" ]]; then
     if [[ ${#SKIP_LANGS[@]} -ne 0 ]]; then
         printf -v ignore_files "/^.\/public\/%s/," "${SKIP_LANGS[@]}"; ignore_files="${ignore_files%,}"
     fi
-    if [[ ${SKIP_EXTERNAL_LINK_CHECK:-} == "true" ]]; then
+    if [[ ${SKIP_EXTERNAL_LINK_CHECK:-} != "false" ]]; then
```

Comment:
I've changed the default behaviior to not check external links (only internal ones). I think we spend too much time building and rebuilding because of temporary issues with external links. IMO, we should just do the full external link check periodically, and not with every build.